docs/spl-docs/stats-command.md

```diff
@@ -0,0 +1,96 @@
+
+**Complete:**  
+Required syntax is in **bold**.
+
+| stats  
+**( \<stats-agg-term\>... )**  
+[\<by-clause\>]  
+
+### Required Arguments
+
+#### stats-agg-term  
+**Syntax:** \<stats-func\>(\<evaled-field\> | \<wc-field\>) [AS \<wc-field\>]\  
+**Description:** A statistical aggregation function. This function can be applied to an eval expression, or to a single field or multiple fields. Utilize the AS clause to assign the result to a new field with a name you choose. You can use wild card characters in field names.
+
+### Optional Arguments
+
+#### by-clause
+**Syntax:** `BY <field-list>`\
+**Description:** The name of one or more fields to group by. Wildcard characters cannot be used to specify multiple fields with similar names; each field must be individually named. The BY clause produces one row for each distinct value in the specified fields. Without a BY clause, the stats command returns a single row, aggregating across the entire incoming results set.\
+
+### Stats function options
+
+#### stats-func
+
+**Syntax:** The syntax depends on the function that you use. Refer to the table below.\
+**Description:** Statistical and charting functions that you can use with the stats command. Each time you invoke the stats command, you can use one or more functions. However, you can only use one BY clause.
+
+The following table lists the supported functions by type of function. Use the links in the table to see descriptions and examples for each function
+
+| Type of function    | Supported functions           |
+|---------------------|-------------------------------------------|
+| [Aggregate functions](./aggregate-functions/agg-functions.md) | `avg()`, `min()`, `max()`, `range()`, `sum()`, `count()`, `distinct_count()` |
+
+## Usage
+
+### Eval expressions with statistical functions
+
+When you use the stats command, you can use an `eval` expression as part of the statistical function. For example:
+
+``` spl
+... | stats count(eval(error_code=503)) AS error_503_count BY endpoint
+```
+
+
+### Wildcards in BY clauses
+
+The stats command does not allow the use of wildcard characters in field values within BY clauses.  
+
+For example, you cannot specify:
+
+```spl
+| stats count BY app*.
+```
+
+### Renaming fields
+
+You cannot assign multiple new names to a single field. For instance, if you have field A, you cannot rename A to both B and C. The following example is not valid:
+
+```spl
+... | stats count(user) AS total_users, count(user) AS unique_users
+```
+
+## Examples
+
+### 1. Calculate the average load time for each server
+
+    ```spl
+    ... | stats avg(load_time) BY server
+    ```
+
+### 2. Calculate the average count for each hour for similar fields using wildcard characters
+
+    Compute the average, for each hour, of any unique field that ends with the suffix "count". For instance, pagecount, errorcount, visitcount, etc.
+
+    ```spl
+    ... | stats avg(*count) BY date_hour
+    ```
```

Comment:
This isn't working
<img width="842" alt="image" src="https://github.com/user-attachments/assets/d3128a91-8aec-4543-ab6b-5156d742d526">
However,
<img width="842" alt="image" src="https://github.com/user-attachments/assets/a8b2bb7f-4426-4193-b531-dc1d989394a6">

Reply:
I removed this example since we do support using wildcard in field names yet.